src/components/AppHeader/AppHeader.css

```diff
@@ -2,6 +2,8 @@
 
 .appBar {
   background-color: var(--backgroundAltColor);
+  height: auto;
+  padding: 8px 2.4rem;
```

Comment:
Yep, the idea is:
1. Use pixels everywhere for now - simple and declarative.
2. When we get an actual design system in place and get rid of all the hodge podge, bring in the ems and rems. It's more complex, but the benefits are there if we implement properly.